documentation/modules/auxiliary/scanner/redis/redis_server.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+Redis is an in-memory data structure project implementing a distributed, in-memory key-value database with optional durability. Redis supports different kinds of abstract data structures, such as strings, lists, maps, sets, sorted sets, HyperLogLogs, bitmaps, streams, and spatial indexes.
+
+This module locates Redis endpoints by attempting to run a specified Redis command.
+
+## Vulnerable Application
+
+This module is tested on two different Redis server instances.
+Virtual testing environments(inside docker container): 
+- Redis 5.0.6
+- Redis 4.0.14
+
+
+## Verification Steps
+
+  1. Do: `use auxiliary/scanner/redis/redis_server`
+  2. Do: `set rhosts [ips]`
+  3. Do: `set command [redis command]`
+  4. Do: `run`
+
```

Comment:
Please add the options header, and include the command option. Make sure to label the default!
If there is a website which lists valid commands, that would be good to link here

Reply:
I have no time now but I'll do in a few days. Thank you for your review.